### DISCUSSION TITLE
Meeting finder not defaulting to the current day

### DISCUSSION THREAD

**Discussion by lbardiv** [code4recovery/12-step-meeting-list#1005] [Q&A] [answered]
We're having an issue on some computers where the day doesn't default to the current day. Also, when the day is changed, the meeting information doesn't update. The weird thing is that it doesn't happen on every computer all the time. My phone app (Android) and computer (MS) tend to work all the time (only had two instances where the day was not current). Our intergroup office's computer (MS) will never have the default day.
Is anyone else having this same issue?
lbardiv@gmail.com

**Comment by joshreisner** [comment, marked as answer]
there are two things that could cause this: 

1) do you have a caching plugin installed? that can make the meeting finder show the previous day (or earlier). one way to address this is to uninstall the plugin, or force the cache somehow to refresh.

2) is the wordpress timezone set correctly?

we might be able to say more if you share your website URL with us

**Reply by lbardiv** [reply]
www.ndiaa.org
Thanks for replying. Should I wait until you take a look before I do anything?

**Reply by joshreisner** [reply]
thanks. i'm seeing Friday meetings (on a Wednesday) so i'm guessing it's a caching plugin showing us the cache from last Friday

another option to fix it would be to switch over to the TSML UI appearance (looks like this: https://sussexaa.org/meetings/) -- this uses the browser's time to show you meetings starting at now

**Comment by lbardiv** [comment]
Thanks! So I'm reading the configuration and on our dashboard, I'm not
seeing meetings. Would you be able to help me?
12 Step Meeting List

If you are using the 12 Step Meeting List plugin
<https://wordpress.org/plugins/12-step-meeting-list/> for WordPress, setup
is simple. Go to Dashboard > Meetings > Import & Settings > Settings and
change User Interface Display → TSML UI

**Reply by joshreisner** [reply]
sure, see this screenshot:

![tsml-ui](https://user-images.githubusercontent.com/1551689/208956643-154b4725-69eb-4a9e-9eea-e70f8c4a5777.png)

oops, i see you found it. well in case it helps someone else, i'll upload anyway

**Comment by lbardiv** [comment]
Please disregard - I found it and am in the process of adding the UI
instructions. Thanks!

**Reply by joshreisner** [reply]
TSML UI is working for me now on your page, i'm seeing the next meeting at 12p wednesday 🎉

**Comment by lbardiv** [comment]
Thank you so much!
